pkgs/development/ocaml-modules/nocrypto/default.nix

```diff
@@ -1,28 +1,37 @@
-{ stdenv, fetchzip, ocaml, findlib, cstruct, type_conv, zarith, ounit }:
+{ stdenv, buildOcaml, fetchFromGitHub, ocaml, findlib, cstruct, type_conv, zarith, ounit, ocaml_oasis, ppx_sexp_conv
```

Comment:
type_conv can be dropped (it was there to get a sexplib with camlp4 support, which is no longer needed)... but sexplib is needed for nocrypto (which might be there due to transitive dependencies (i.e. cstruct depends on sexplib already))